src/context/AuthContext.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+
+const defaultContext = {
+  isLoggedIn: false,
+  login: () => {},
+  logout: () => {},
+};
+const AuthContext = React.createContext(defaultContext);
+
+const AuthContextProvider = ({ children }) => {
+  const [isLoggedIn, setIsLoggedIn] = React.useState(
+    !!localStorage.getItem('isLoggedIn')
+  );
+
+  const login = () => {
+    localStorage.setItem('isLoggedIn', true);
+    const token = localStorage.getItem('isLoggedIn');
+    setIsLoggedIn(token);
```

Comment:
```suggestion
    //const token = localStorage.getItem('isLoggedIn');
    setIsLoggedIn(true);
```

localStorage에서 getItem의 리턴된 값을 넣어주는 게 아니라 바로 setIsLoggedIn에 `true`로 넣어주는 건 어떨까요? (코드 한 줄 줄이기 효과)